src/validations/multi-city-trip.validation.js

```diff
@@ -58,22 +59,24 @@ export default async (req, res, next) => {
     ResponseService.setError(400, allErrorMessages);
     return ResponseService.send(res);
   }
-  let tripId = `${req.userData.id}${new Date().getFullYear()}${new Date().getMonth()}${new Date().getDate()}`;
-  req.body.forEach(trip => {
-    tripId += `${trip.destinationId}`;
-  });
   const filteredTrips = req.body.filter((trip, index) => index !== 0);
   filteredTrips.forEach((trip, index) => {
     if (req.body[index + 1].originId !== req.body[index].destinationId) {
       ResponseService.setError(400, 'Next origin location have to be the same as previous destination location');
       return ResponseService.send(res);
     }
   });
-  const tripExist = await RequestService.findTripByProperty({ tripId });
-  if (tripExist) {
+  const trips = await Promise.all(req.body.map(async (trip) => {
+    const tripExist = await TripService.findTripByProperty({
```

Comment:
Use the `IN` operator at the database level instead of .map to find multiple trips